bokun.yaml

```diff
@@ -5537,413 +5734,535 @@ definitions:
+        description: 'The number of days before the activity when we stop accepting bookings.'
       bookingCutoffWeeks:
         type: integer
         format: int32
         description: 'The number of weeks before the activity when we stop accepting bookings.'
-      box:
-        type: boolean
-        description: 'Specifies whether this is a boxed version of another product.'
+      requestDeadlineMinutes:
+        type: integer
+        format: int32
+        description: 'The number of minutes allocated to confirm or reject booking by activity supplier.'
+      requestDeadlineHours:
+        type: integer
+        format: int32
+        description: 'The number of hours allocated to confirm or reject booking by activity supplier.'
+      requestDeadlineDays:
+        type: integer
+        format: int32
+        description: 'The number of days allocated to confirm or reject booking by activity supplier.'
+      requestDeadlineWeeks:
+        type: integer
+        format: int32
+        description: 'The number of weeks allocated to confirm or reject booking by activity supplier.'
       boxedActivityId:
         type: integer
         format: int64
         description: 'If this is a box, then this field holds the ID of the activity inside the box.'
-      boxedVendor:
-        $ref: '#/definitions/VendorDto'
-        description: 'If this is a box, then this field holds the ID of the vendor that owns the activity inside the box.'
-      categories:
-        type: array
-        items:
-          $ref: '#/definitions/CategoryDto'
       comboActivity:
         type: boolean
         description: 'Specifies whether this is a combo activity.'
-      paymentCurrencies:
+      comboParts:
         type: array
         items:
-          type: string
-      customDropoffAllowed:
+          $ref: '#/definitions/ComboPart'
+        description: 'Deprecated. The list of the components of this combo product.'
+      ticketPerComboComponent:
         type: boolean
-        description: 'Only relevant if dropoffService is TRUE. Specifies whether the customer should be allowed to enter their drop off place in free text.'
-      customFields:
+        description: 'Deprecated. Specifies whether this combo activity has tickets per combo component.'
+      ticketComboComponents:
         type: array
         items:
-          $ref: '#/definitions/CustomFieldDto'
-      customPickupAllowed:
+          $ref: '#/definitions/ComboPart'
+          description: 'Deprecated. The list of the parts of this combo product for which customer gets separate tickets as well as a ticket for the combo itself.'
+      pickupActivityId:
+        type: integer
+        format: int64
+        description: 'Deprecated.'
+      allowCustomizedBookings:
         type: boolean
-        description: 'Only used if meetingType != MEET_ON_LOCATION. Specifies whether the customer should be allowed to enter their pick up place in free text.'
+        description: 'Deprecated.'
```

Comment:
Why is this deprecated?

Reply:
I found only one usage of this variable in the code, only "value write", so decided that it is not used anymore.

Reply:
Btw, are we speaking about "pickupActivityId"?

Reply:
Tadas could not help with this to start working also.

Reply:
Found how it works. Documented.